src/core/tests/frontend/onnx/CMakeLists.txt

```diff
@@ -17,10 +17,14 @@ install(TARGETS ${TARGET_NAME}
         COMPONENT tests
         EXCLUDE_FROM_ALL)
 
-set(TEST_ONNX_MODELS_DIRNAME ${TEST_MODEL_ZOO}/src/core/models/onnx)
+set(TEST_ONNX_MODELS_DIRNAME ${TEST_MODEL_ZOO}/core/models/onnx)
 target_compile_definitions(${TARGET_NAME} PRIVATE -D TEST_ONNX_MODELS_DIRNAME=\"${TEST_ONNX_MODELS_DIRNAME}/\")
 set(MANIFEST ${CMAKE_CURRENT_SOURCE_DIR}/unit_test.manifest)
 target_compile_definitions(${TARGET_NAME} PRIVATE -D MANIFEST=\"${MANIFEST}\")
 
 add_dependencies(${TARGET_NAME} ov_onnx_frontend)
 add_dependencies(${TARGET_NAME} test_model_zoo)
+
+# Add include path to so_extension.hpp
+set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/conversion.cpp
+        PROPERTIES INCLUDE_DIRECTORIES "${OpenVINO_SOURCE_DIR}/src/core/src/")
```

Comment:
@ilyachur let's add `dev_api` for `core`

Reply:
@ilyachur do we have a ticket for it?

Reply:
Added dev_api in the PR: https://github.com/openvinotoolkit/openvino/pull/9571

Reply:
merged

Reply:
fixed